.eslintignore

```diff
@@ -3,3 +3,6 @@ src/**/doc/*
 build/
 docs/
 protos/
+test/fixtures/fib.js
```

Comment:
Instead of calling these out individually, add `test/fixtures` and ignore the whole dir

Reply:
Added
```
test/fixtures
system-test/fixtures
```
Thanks